memory/database.go

```diff
@@ -227,14 +228,41 @@ func (d *BaseDatabase) AddTable(name string, t MemTable) {
 	d.tables[name] = t
 }
 
+func BacktickDefaultColumnValueNames(sch sql.Schema) sql.Schema {
```

Comment:
Why is this here? Put this in the rowexec package?

Reply:
putting this in the rowexec package results in an import cycle